### ISSUE TITLE
Getting diff 

### ISSUE THREAD

**Issue by dbadola** [terraform-providers/terraform-provider-fortios#6]
I tried to create object address like below

```
resource "fortios_firewall_object_address" "s4" {
    name = "s4"
    type = "ipmask"
    subnet = "1.1.1.0 255.255.255.0"
    comment = "dd"
}
```

once I applied the changes using `terraform apply` I can see it's being pushed to the firewall.

Now the problem is on the next run without doing any changes when I do `terraform plan` I am getting `update in-place` output which I should not get

```
------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ fortios_firewall_object_address.s4
      end_ip:   "255.255.255.0" => "0.0.0.0"
      start_ip: "1.1.1.0" => "0.0.0.0"


Plan: 0 to add, 1 to change, 0 to destroy.

------------------------------------------------------------------------
```

**Comment by fwiethof** [comment]
I can confirm this also happens here.

**Comment by fwiethof** [comment]
@frankshen01 that resolved the issue.

However, there is another issue with prefix length / subnet mask in address objects.
See https://github.com/terraform-providers/terraform-provider-fortios/issues/13

**Comment by frankshen01** [comment]
This has been released in version 1.1.0 of the Terraform FortiOS provider.
